sync.py

```diff
@@ -117,25 +119,31 @@ def yes_or_no(question):
 	if choice:
 		print "All right, will sync anyway."
 		wrong_perm = True
-	else: 
+	else:
 		print "All right, aborting."
 		sys.exit()
 
-else: 
+else:
 	print "Everything looks OK. Continuing with rsync."
 
 
 # Step 5. Sync our destignated folders.
 
-excludes = "--exclude=*.swp --exclude=irma3/"
-rsync_cmd = "/bin/rsync -avzP --omit-dir-times {0} --log-file={1} {2} {3}@{4}:{5}".format(excludes, rsync_log_path, src_root_path, user, host, dest)
-# TODO: Do this cleaner? 
-dry_cmd = "/bin/rsync --dry-run -avzP --omit-dir-times {0} {1} {2}@{3}:{4}".format(excludes, src_root_path, user, host, dest)
-
-# First doing a dry-run to confirm sync. 
+excludes = "--exclude=*.swp --exclude=irma3/ --exclude='*'"
+rsync_cmd = "/bin/rsync -avzP --omit-dir-times {0} --log-file={1} {2} {3} {4}@{5}:{6}".format(excludes,
+																								rsync_log_path,
```

Comment:
Woow, indentation looks really weird here. Maybe tabs is used instead of spaces?